plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -490,9 +492,12 @@ class read_only {
 
          if( p.lower_bound.size() ) {
             if( p.key_type == "name" ) {
-               name s(p.lower_bound);
-               SecKeyType lv = convert_to_type<SecKeyType>( s.to_string(), "lower_bound name" ); // avoids compiler error
-               std::get<1>(lower_bound_lookup_tuple) = conv( lv );
+               if constexpr (std::is_same_v<uint64_t, SecKeyType>) {
```

Comment:
@larryk85 @b1bart I'm showing up a little late to this, but this sure doesn't seem like what we want.  If "name" means "expect a uint64_t" why don't we just call it "uint64_t".  I think what we really want is something that checks to see if the parameter has any letters, '.', or just 1-5 and we use string, otherwise we take it as a uint64_t.  For get_table_rows_ex, "name" is still treated as a string to be passed to a name{}, and I don't think it makes sense to handle it 2 different ways.

Reply:
Need to resolve this, since both of the mentioned interfaces effect the RPC code in working on

Reply:
I don't know if I just got confused by the comments that I read from earlier, or if my brain isn't working, because this is correct. I thought from the discussions that this was just testing everything as uint64.